### PR TITLE
Add kafka client

### PR DESCRIPTION
**Pull request by NV4RE**
- worker client
 - start transaction (kafka)

### CODE REVIEW

File: kafka_client.go [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		TaskID:        t.TaskID,
+		Status:        t.Status,
+		Output:        t.Output,
+		Logs:          []interface{}{},
+		DoNotRetry:    false,
+	}
+}
+
+func NewWorkerClient(kafkaServers string, namespace string, kafkaVersion string) (*kafkaClient, goerror.Error) {
+	ks := strings.Split(kafkaServers, ",")
```
Comment (zier) [inline kafka_client.go]: please move this line to before line that use this variable  ```ks```

**Comment by NV4RE** [comment]
@zier Here you are.

File: kafka_client.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
+package melonade_client_go
+
+import (
+	"context"
+	"encoding/json"
```
Comment (zier) [inline kafka_client.go:5]: use this https://github.com/json-iterator/go instead standard lib

File: kafka_client.go [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+		return nil, goerror.DefineInternalServerError("UnableToCreateProducer", fmt.Sprint(err))
+	}
+
+	return &kafkaClient{namespace, ks, config, pd}, nil
+}
+
+func (w *kafkaClient) NewWorker(taskName string, tcb func(t *Task) *taskResult, ccb func(t *Task) *taskResult) goerror.Error {
+	c, err := sarama.NewConsumerGroup(w.kafkaServers, fmt.Sprintf(`melonade-%s.client`, w.namespace), w.config)
+	if err != nil {
+		return goerror.DefineInternalServerError("UnableToCreateConsumer", fmt.Sprint(err))
```
Comment (zier) [inline kafka_client.go]: can you define error in errors.go?

**Comment by NV4RE** [comment]
@zier Fixed

**Pull request merged**